includes/API/GetPageByRevIdHandler.php

```diff
@@ -69,7 +69,7 @@ public function getParamSettings() {
 	protected function provideTitle( int $revId ): ?Title {
 		$revisionRecord = $this->revisionLookup->getRevisionById( $revId );
 		if ( !$revisionRecord ) {
-			throw new HttpException( "invalid rev_id", 404 );
+			throw new HttpException( "Not found", 404 );
```

Comment:
@pwirth the error message needs to be made consistent. While this is 404, it is different from https://github.com/inblockio/DataAccounting/blob/4f21415e5d8fa966e3e8beea5a8b986b81cd20ae/includes/API/GetPageByRevIdHandler.php#L25.

Reply:
We can make them to be just "Not found" to make it easier to be consistent.

Reply:
i added the missing 404 http status. I think, to have an actual error status here is essential